locale/en/auth.ftl

```diff
@@ -0,0 +1,314 @@
+codes-generate = Generate codes
+codes-generate-plaintext = { codes-generate }:
+lowRecoveryCodes-action = Generate codes
+lowRecoveryCodes-subject =
+    { $numberRemaining ->
+        [one] 1 recovery code remaining
+       *[other] { $numberRemaining } recovery codes remaining
+   }
+
+# Variables:
+#  $clientName (String) - A client the user hasn't signed into before (e.g. Firefox, Sync)
+newDeviceLogin-subject = New sign-in to { $clientName }
+newDeviceLogin-title = New sign-in to { $clientName }
+newDeviceLogin-action = Manage account
+
+passwordChanged-subject = Password updated
+passwordChanged-title = Password changed successfully
+passwordChanged-description = Your { -product-firefox-account } password was successfully changed from the following device:
+
+passwordChangeRequired-subject = Suspicious activity detected
+passwordChangeRequired-title = Password Change Required
+passwordChangeRequired-suspicious-activity = We detected suspicious behavior on your { -product-firefox-account }. To prevent unauthorized access to your { -product-firefox-account }, we’ve disconnected all devices on your account and are requiring you to change your password as a precaution.
+passwordChangeRequired-sign-in = Sign back into any device or service where you use your { -product-firefox-account } and follow the steps that will be presented to you.
+passwordChangeRequired-different-password = <b>Important:</b> Pick a different password than what you were previously using and make sure that it is different from your email account.
+passwordChangeRequired-signoff = Best,
+passwordChangeRequired-signoff-name = The { -product-firefox-accounts } team
+passwordChangeRequired-different-password-plaintext = Important: Pick a different password than what you were previously using and make sure that it is different from your email account.
+
+passwordReset-subject = Password updated
+passwordReset-title = Your account password was changed
+passwordReset-description = You will need to enter your new password on other devices to resume syncing.
+
+passwordResetAccountRecovery-subject = Password updated using recovery key
+passwordResetAccountRecovery-title = Your account password was reset with a recovery key
+passwordResetAccountRecovery-description = You have successfully reset your password using a recovery key from the following device:
+passwordResetAccountRecovery-action = Create new recovery key
+passwordResetAccountRecovery-regen-required = You will need to generate a new recovery key.
+# After the colon, there's a link to https://accounts.firefox.com/settings/account_recovery
+passwordResetAccountRecovery-create-key = Create new recovery key:
+
+postAddAccountRecovery-subject = Account recovery key generated
+postAddAccountRecovery-title = Account recovery key generated
+postAddAccountRecovery-description = You have successfully generated an account recovery key for your { -brand-firefox } Account using the following device:
```

Comment:
I assume we want to fix all existing instances of brand-firefox + account

```suggestion
postAddAccountRecovery-description = You have successfully generated an account recovery key for your { -product-firefox-account } using the following device:
```

Reply:
Oh yes, this was missed, thanks for the catch.